### DISCUSSION TITLE
Can I turn off the For You feature?

### DISCUSSION THREAD

**Discussion by mantoufan** [community/community#14222] [Other Feature Feedback, Questions, & Ideas]
I don't like this feature -- For You, it shows me people I don't have any cooperation with anymore, and I would be recommended to those people. It looks very awkward and bad.
It's like, you know, one day, you see your ex-boyfriend, or your ex-girlfriend, and what's the point of that other than being sad?
If I want to see what's going on with someone or a project, I'll click on Star, Watch or Follow button, and that's good enough for me.
If you insist on this feature, **Provide a button to turn off this feature, pls**
- When closed, I don't see the For You TAB anymore
- When closed, I wouldn't be recommended to anyone else

This is very important! Please consider our proposal carefully. Thank you very much!

**Comment by Joelsamijon** [comment]
I apologize for that man,yah just go ahead..iif I make and help you comfortable..sure.

Sent from Yahoo Mail on Android 
 
  On Sun, Apr 3, 2022 at 8:50 PM, ***@***.***> wrote:   


I don't like this feature -- For You, it shows me people I don't have any cooperation with anymore, and I would be recommended to those people. It looks very awkward and bad.
It's like, you know, one day, you see your ex-boyfriend, or your ex-girlfriend, and what's the point of that other than being sad?
If I want to see what's going on with someone or a project, I'll click on Star, Watch or Follow button, and that's good enough for me.
If you insist on this feature, please provide us with an opt-out or opt-out option.
   
   - When closed, I don't see the For You TAB anymore
   - When closed, I wouldn't be recommended to anyone else   
This is very important! Please consider our proposal carefully. Thank you very much!

**Comment by Emargitza33** [comment]
[Uploading 2021-GitHub-SOC-3-Report.pdf…]()

**Comment by killshot13** [comment]
How is either of these answers relevant? Neither of them even expresses a complete thought, much less provides a real solution.

**Reply by mantoufan** [reply]
Sorry, I've unmarked the answer, I don't know who is official staff and who is going to help us with our feedback.
We need a bit of a practical solution.

**Reply by killshot13** [reply]
Agreed @mantoufan, also I wasn't directing my comment at you so much as the authors. Still, I appreciate the unmark. I plan to check some of the other threads to see how guidance is faring elsewhere on this topic.

**Comment by abelcheung** [comment]
Though it seems Github developers are quite determined to push this to every users at all costs, users can still block it in their browsers.

Somebody has created a Chrome extension [just for this purpose](https://github.com/github/feedback/discussions/13154#discussioncomment-2420337) (EDIT: [another extension for Firefox](https://github.com/github/feedback/discussions/13130#discussioncomment-2472016)). Ad blocker with custom rule can also block the feed from displaying. It is still loaded in the background, but at least you can get back some peace of mind by hiding it.

This is the ruleset for uBlock, for example:

    github.com##.js-for-you-feed
    github.com###feed-next

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by mantoufan** [reply]
Thanks for your work. I canceled all checks.
I'd like to know how to avoid appearing in others' _For you_

**Reply by jfullstackdev** [reply]
make your profile private by going here

https://github.com/settings/profile

then at the bottom 

![image](https://user-images.githubusercontent.com/47092464/182022338-614507cc-8d39-4321-aa66-e65d4964086d.png)

**Reply by mantoufan** [reply]
Thank you, but I just want my information not to appear in others' _For you_, rather than hide my information.
To be more specific, it's like before _For you_ went online.
> 1. Users who have followed me and visited my home page can see my information  
> 2. Users who don't follow me, only those who have contact with me, can't see my information through _For you_

This is because those people don't pay attention to me, and we don't cooperate anymore. 
**It's embarrassing for us to see each other's information in _For you_**.
It's like suddenly seeing the information of my ex-girlfriend or ex-boyfriend's on GitHub.
So far, _For you_ has not provided a function that allows me to disable it completely, at least not to let my information appear in my ex-girlfriend or ex-boyfriend's _For you_

> Maybe we can give the option back to users. We can decide who we want to see by clicking _Follow_.

**Reply by jfullstackdev** [reply]
then block the user

**Reply by mantoufan** [reply]
Block user is easy to cause misunderstanding. 
We just don't want to see each other's information in _For you_, rather than completely stop each other.
Blcok user means breaking up with each other in our culture. 
Few people will be willing to do this unless they have deep hatred for each other or block advertising.

**Reply by mantoufan** [reply]
I can see my previous boss's dynamic in _For you_.
I believe that my boss can also see my dynamics.
Blocking the previous boss is obviously inappropriate, but letting the previous boss see what I'm doing at my current company doesn't seem appropriate either.

An option that only works for _For you_ is:
> Block recommend me to anyone at _For you_

That's good enough for me.

If he wants to come to my homepage to follow me, of course it's fine.
But I don't want my information to be actively displayed on his _For you_, making him feel uncomfortable.

Thank you for your attention and improvement to user feedback.
